src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -1,77 +1,52 @@
 package com.kodilla.ecommercee.controller;
 import com.kodilla.ecommercee.domain.carts.Cart;
+import com.kodilla.ecommercee.domain.orders.Order;
 import com.kodilla.ecommercee.domain.orders.OrderDto;
 import com.kodilla.ecommercee.domain.orders.ItemDto;
 import com.kodilla.ecommercee.domain.users.User;
+import com.kodilla.ecommercee.mapper.OrderMapper;
+import com.kodilla.ecommercee.repository.OrderRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.Date;
 import java.util.List;
+import java.util.Optional;
 
 @RestController
 @RequestMapping("/v1/order")
 public class OrderController {
-    //lista wszystkich zamówień ok
-    //dodanie nowego zamówienia ok
-    //wyświetlenie zamówienia ok
-    //aktualizacja zamówienia ok
-    //usunięcie zamówienia ok
-
-    List<OrderDto> orders = Collections.emptyList();
+    @Autowired
+    private OrderRepository orderRepository;
+    @Autowired
+    private OrderMapper orderMapper;
 
     @GetMapping("/getOrders")
     private List<OrderDto> getOrders() {
-        return getOrderDtos();
+        return orderMapper.orderListToOrderDtoList(orderRepository.findAll());
     }
 
     @PostMapping("/createOrder")
     private void createOrder(@RequestBody OrderDto orderDto){
-        orders.add(orderDto);
+        orderRepository.save(orderMapper.orderDtoToOrder(orderDto));
     }
 
     @GetMapping("/getOrder")
     private OrderDto getOrder(@RequestParam("orderId") long orderId){
-        getOrderDtos();
-        for (OrderDto order:orders) {
-            if(order.getOrderId() == orderId){
-                return order;
-            }
-        }
-        return null;
+        Optional<Order> order = orderRepository.findById(orderId);
+        return orderMapper.orderToOrderDto(order.get());
```

Comment:
Rozważyłbym tutaj rzucenie wyjątkiem w stylu: OrderNotFoundException :)

Reply:
Ogólnie to powinno być mniej więcej tak: 
```
orderRepository.findById(orderId)
   .map(orderMapper::orderToOrderDto)
   .orElseThrow(OrderNotFoundException::new)
```

Reply:
Poprawione